src/vs-bicep/Bicep.VSLanguageServerClient/DidChangeWatchedFilesNotifier.cs

```diff
@@ -22,7 +22,7 @@ namespace Bicep.VSLanguageServerClient
     /// </summary>
     public class DidChangeWatchedFilesNotifier : IDisposable
     {
-        private readonly string[] filters = ["*.arm", "*.bicep", "*.json", "*.jsonc"];
+        private readonly string[] filters = ["*.arm", "*.bicep", "*.json", "*.jsonc", "*.bicepparam"];
```

Comment:
Nice! Does this solve #12418?

Reply:
Unfortunately no, but I think it might not be a lot more work, will give it a try soon.  This shouldn't hurt anything though.